docs/EN_US/ContainerizedHPCC/ContainerizedMods/RemoteAccess.xml

```diff
@@ -0,0 +1,399 @@
+
+          <para>(where mycustomsettings.yaml contains all values and changes
+          discussed above)</para>
+
+          <para>At this point the HPCC cluster named env1 starts. In doing so,
+          the "remote" certificate authority generates the client certificates
+          defined in the step above.</para>
+
+          <para>The HPCC cluster named env1 is now setup, but you must extract
+          information from it to supply to the client environments (in this
+          example -- env2).</para>
+        </listitem>
+
+        <listitem>
+          <para>Extract the client secret certificate(s) and key pair(s) from
+          the deployed cluster.</para>
+
+          <para><programlisting>kubectl get secrets</programlisting></para>
+
+          <para>All those prefixed with <emphasis
+          role="bold">client-remote-</emphasis> and <emphasis
+          role="bold">client-remote-dfs-</emphasis> are the secrets to use for
+          the client(s) configured earlier.</para>
+
+          <para>Extract the certificate and key pairs and save to files with
+          these commands:</para>
+
+          <variablelist>
+            <varlistentry>
+              <term>Note:</term>
+
+              <listitem>
+                <para>You may may meed to install <emphasis
+                role="bold">jq</emphasis> (a command line JSON processor).
+                </para>
+              </listitem>
+            </varlistentry>
+          </variablelist>
+
+          <para><programlisting><code>kubectl get secret client-remote-dfs-dfs-env2-tls -o \
+            json | jq -r '.data."tls.crt"'| base64 -d &gt; env2tls.crt 
+
+kubectl get secret client-remote-dfs-dfs-env2-tls -o \
+            json | jq -r '.data."tls.key"'| base64 -d &gt; env2tls.key 
+
+kubectl get secret client-remote-dfs-dfs-env2-tls -o \
+            json | jq -r '.data."ca.crt"' | base64 -d &gt; env2ca.crt </code></programlisting></para>
+
+          <para><emphasis role="bold">Keep a record of the files created here,
+          they are needed when configuring the client
+          environment.</emphasis></para>
+        </listitem>
+
+        <listitem>
+          <para>Get the DFS service external endpoint:</para>
+
+          <para><programlisting>kubectl get svc</programlisting></para>
+
+          <para>Example output:</para>
+
+          <para><programlisting>NAME                 TYPE          CLUSTER-IP    EXTERNAL-IP    PORT(S)         AGE 
+cert-manager         ClusterIP     10.0.1.53     &lt;none&gt;         9402/TCP        24h 
+cert-manager-webhook ClusterIP     10.0.13.136   &lt;none&gt;         443/TCP         24h 
+dfs                  LoadBalancer  10.0.108.249  <emphasis role="red">192.168.0.36   8520</emphasis>:32469/TCP  23h </programlisting></para>
+
+          <para><emphasis role="bold">Keep a record of the external dfs
+          service endpoint, in this example, it’s
+          192.168.0.36:8520.</emphasis></para>
```

Comment:
I think it would be better not to use IP addresses in examples especially when it comes to our TLS/certificates examples.
You'll find that lately the environments being set up internally have domains and fqdns for each service.

For example the dfs for one of our clusters would be something like this made up example:

dfs.us-productnamethor-dev.azure.ourdomain.io

It's important to document this way because you also have to set the certificates.issuers.remote.domain value.  

When setting up our first real remote access environments they initially didn't know to add the domain, and it's a very important piece.

Setting that means both the server and remote client certificates will correctly identify the domain and service being accessed.
```
    remote:
      domain: us-productnamethor-dev.azure.ourdomain.io
      enabled: true
      kind: Issuer
      name: hpcc-remote-issuer
      spec:
        ca:
          secretName: hpcc-remote-issuer-key-pair

```